scalableminds/webknossos
# Hide unreported datasets

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- backend: dataset json api should include new field `isUnreported` (true iff datastores unreported this dataset (“no longer avaible on the datasetore”)
- frontend:
  - log in as admin and use the filtering settings next to the search box in the datasets tab
  - log in as non-admin. the settings box should be hidden. as before, the table should only show datasets which have datalayers.

### Issues:
- first part of #3773 

------
